### PR TITLE
Fix mongo shard migration

### PR DESCRIPTION
**Pull request by 123MwanjeMike**
**_WHAT DOES THIS PR DO?_**
- Fixes mongo shard migration issues

**_WHAT ISSUES ARE RELATED TO THIS PR?_**

**_HOW DO I TEST OUT THIS PR?_**

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**

**_ARE THERE ANY RELATED PRs?_**

### CODE REVIEW

File: infra/ansible/uninstall-mongo.yml
Comment (123MwanjeMike) [inline infra/ansible/uninstall-mongo.yml:1]: [uninstall mongodb playbook](https://github.com/airqo-platform/AirQo-api/pull/1538/commits/af033721255ff0262fcdc7c9659c70a14d0123d8)

File: infra/ansible/k8s_preemptive_nodes.yml
Comment (123MwanjeMike) [inline infra/ansible/k8s_preemptive_nodes.yml:1]: [rename playbooks](https://github.com/airqo-platform/AirQo-api/pull/1538/commits/2282a9f959b8b2c43561b66762510a13c88bd18e)

File: infra/ansible/inventories/production/hosts [hunk 1]
```diff
@@ -26,11 +26,21 @@ all:
...
+        mongos_router:
+          ansible_host: mongos-router.europe-west1-b.c.airqo-250220.internal
       children:
         config_servers:
           hosts:
-            mongod-cfgsvr-[1:3].europe-west1-b.c.airqo-250220.internal
-        shards_1:
+            mongod-cfgsvr-0.europe-west1-b.c.airqo-250220.internal:
+            mongod-cfgsvr-1.europe-west1-c.c.airqo-250220.internal:
+            mongod-cfgsvr-2.europe-west1-d.c.airqo-250220.internal:
```
Comment (123MwanjeMike) [inline infra/ansible/inventories/production/hosts:36]: [correct mongodb hosts FQDNs](https://github.com/airqo-platform/AirQo-api/pull/1538/commits/0251259034c812e31955a7b9b5583316750a5c73)

File: infra/ansible/inventories/production/hosts [hunk 2]
```diff
@@ -26,11 +26,21 @@ all:
...
-            mongod-shard-1-[1:3].europe-west1-b.c.airqo-250220.internal
\ No newline at end of file
+            mongod-shard-0-0.europe-west1-b.c.airqo-250220.internal:
+            mongod-shard-0-1.europe-west1-c.c.airqo-250220.internal:
+            mongod-shard-0-2.europe-west1-d.c.airqo-250220.internal:
+        shard_1:
+          hosts:
+            mongod-shard-1-0.europe-west1-b.c.airqo-250220.internal:
+            mongod-shard-1-1.europe-west1-c.c.airqo-250220.internal:
+            mongod-shard-1-2.europe-west1-d.c.airqo-250220.internal:
```
Comment (123MwanjeMike) [inline infra/ansible/inventories/production/hosts:46]: [add shard_1 to hosts](https://github.com/airqo-platform/AirQo-api/pull/1538/commits/edd14d103956eb85bb0bd0ce3bea4becd696ac7f)

**Review by Baalmart — APPROVED** [review]
Hi @123MwanjeMike , thanks.
All looks good to me.

**Pull request merged**